Examples/ExamplesApp/Info.plist

```diff
@@ -10,6 +10,43 @@
 	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
 	<key>CFBundleInfoDictionaryVersion</key>
 	<string>6.0</string>
+	<key>CFBundleLocalizations</key>
```

Comment:
These values referenced from the Cocoa supported languages.

Even though the Examples app itself won't be localized, adding these allows us to test component localization in the app.